src/app/shared/navigation-focus/navigation-focus.ts

```diff
@@ -14,7 +14,7 @@ export class NavigationFocus implements OnInit {
   ngOnInit() {
     clearTimeout(lastTimeoutId);
     // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-    lastTimeoutId = setTimeout(() => this.el.nativeElement.focus(), 100);
+    lastTimeoutId = window.setTimeout(() => this.el.nativeElement.focus(), 100);
```

Comment:
Why the `window` here?

Reply:
Not sure, but I guess I got an error when tried to run a specific command.